ParseTree/ParseTree/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using ParseTree;
+
+string? filePath = Console.ReadLine();
+
+if (File.Exists(filePath))
+{
+    try
+    {
+        Node root = Tree.BuildTree(filePath);
+
+        Console.WriteLine(root.WriteExpression());
+        Console.WriteLine(root.Calculate());
+    }
+    catch (Exception exeption)
+    {
+        Console.WriteLine(exeption.Message);
+    }
+}
+else
+{
+    Console.WriteLine("Incorrect file path");
+}
```

Comment:
```suggestion
if (!File.Exists(filePath))
{
    Console.WriteLine("Incorrect file path");
}
```
Лучше инвертировать, чтобы не создавать лишний уровень вложенности